.circleci/ci/src/jobs/job-test-apim-charts.ts

```diff
@@ -0,0 +1,42 @@
+import { commands, Config, executors, Job, reusable } from '@circleci/circleci-config-sdk';
+import { Command } from '@circleci/circleci-config-sdk/dist/src/lib/Components/Commands/exports/Command';
+import { config } from '../config';
+import { orbs } from '../orbs';
+import { HelmLintCommand, HelmTestCommand } from '../commands';
+
+export class TestApimChartJob {
+  private static jobName = 'job-test-apim-charts';
+
+  public static create(dynamicConfig: Config): Job {
+    dynamicConfig.importOrb(orbs.helm);
+
+    const helmLintCommand = HelmLintCommand.create();
+    dynamicConfig.addReusableCommand(helmLintCommand);
+
+    const helmTestCommand = HelmTestCommand.create();
+    dynamicConfig.addReusableCommand(helmTestCommand);
+
+    const steps: Command[] = [
+      new commands.Checkout(),
+      new reusable.ReusedCommand(orbs.helm.commands['install-helm-client'], {
+        version: 'v3.7.1',
+      }),
+      new commands.Run({
+        name: 'install helm-unittest plugin',
+        command: 'helm plugin install https://github.com/quintush/helm-unittest --version 0.2.11',
```

Comment:
Same here for helm-unittest version

Reply:
same here I don't know if it's really necessary to have it. I would like to let it like that and try to remove it at the end of our migration. Wdyt ?